larq_compute_engine/core/bgemm_kernels_common.h

```diff
@@ -67,42 +63,57 @@ struct BinaryKernelParams {
   std::int32_t rhs_stride;
   std::int32_t dst_stride;
   std::int32_t depth;
-  std::int32_t clamp_min;
-  std::int32_t clamp_max;
-  DstScalar dst_tmp_buf[LhsCols * RhsCols];
+  OutputTransform<DstScalar> output_transform;
```

Comment:
I've modified the kernel params to include the output transform as a member, which makes dealing with different output transforms (with different output types) easier.